packages/module/src/ConsoleShared/src/components/layout/PageLayout.scss

```diff
@@ -1,29 +1,29 @@
 .pfext-page-layout {
   &__content {
-    padding: var(--pf-v5-global--spacer--lg);
+    padding: var(--pf-v6-global--spacer--lg);
     flex: 1;
   }
 
   &__header {
     padding-top: 1px;
-    background-color: var(--pf-v5-global--BackgroundColor--light-100);
+    background-color: var(--pf-v6-global--BackgroundColor--light-100);
   }
 
   &__hint {
-    padding-left: var(--pf-v5-global--spacer--lg);
-    padding-right: var(--pf-v5-global--spacer--lg);
-    padding-bottom: var(--pf-v5-global--spacer--md);
+    padding-left: var(--pf-v6-global--spacer--lg);
+    padding-right: var(--pf-v6-global--spacer--lg);
+    padding-bottom: var(--pf-v6-global--spacer--md);
     font-size: 14px;
   }
 
   &__title {
-    padding-left: var(--pf-v5-global--spacer--lg);
-    padding-right: var(--pf-v5-global--spacer--lg);
-    font-size: var(--pf-v5-global--FontSize--2xl);
+    padding-left: var(--pf-v6-global--spacer--lg);
+    padding-right: var(--pf-v6-global--spacer--lg);
+    font-size: var(--pf-v6-global--FontSize--2xl);
```

Comment:
There are heading-level specific font-sizes here - https://github.com/patternfly/patternfly/blob/ceb0aa1a538064c928727d7051792bf07ca6ff78/src/patternfly/base/tokens/tokens-default.scss#L369-L374

If that doesn't work for the role of this title, the t-shirt size font-sizes are here - https://github.com/patternfly/patternfly/blob/ceb0aa1a538064c928727d7051792bf07ca6ff78/src/patternfly/base/tokens/tokens-default.scss#L228-L235